spec/integration_spec.rb

```diff
@@ -35,26 +35,10 @@ def phctl(*args)
       end
     end
 
-    it "HtItems creates clusters with ocns in hathifile" do
-      expect { phctl("load", "ht_items", fixture("hathifile_sample.txt")) }
-        .to change { Cluster.count }.by(5)
-    end
-
-    it "Concordance loads concordance diffs" do
```

Comment:
Functionality removed for now since we don't need to add concordance rules to clusters; we can look at how to load the deltas to mariadb given other functionality we already have